fmriprep/workflows/epi.py

```diff
@@ -289,12 +289,16 @@ def normalize_motion_func(in_file, format):
 
     gen_ref = pe.Node(EstimateReferenceImage(), name="gen_ref")
 
+    cphdr = pe.Node(CopyHeader(), name='cphdr')
+
     workflow.connect([
         (inputnode, gen_ref, [('epi', 'in_file')]),
         (gen_ref, inu, [('ref_image', 'input_image')]),
-        (inu, hmc, [('output_image', 'ref_file')]),
-        (inu, skullstrip_epi, [('output_image', 'in_file')]),
-        (inu, outputnode, [('output_image', 'ref_image')]),
+        (inu, cphdr, [('output_image', 'in_file')]),
+        (gen_ref, cphdr, [('ref_image', 'hdr_file')]),
+        (gen_ref, hmc, [('ref_image', 'ref_file')]),
+        (cphdr, skullstrip_epi, [('out_file', 'in_file')]),
+        (cphdr, outputnode, [('out_file', 'ref_image')]),
```

Comment:
Really? Do they give a reason, or is this a bug we should report? And is this the primary use for the `CopyHeader` interface? If so, maybe we should make a reusable workflow that is simply `N4BiasFieldCorrection` + `CopyHeader` so that it's obvious why we're doing this.